che-incubator/chectl
# feat: enhance post installation messages

Signed-off-by: Anatolii Bazko <abazko@redhat.com>



### What does this PR do?
Improve post installation messages

### Screenshot/screencast of this PR
```
  ✔ Show important messages
    ✔ [ACTION REQUIRED] Please add Che self-signed CA certificate into your browser: /home/tolusha/cheCA.crt.
   Documentation how to add a CA certificate into a browser: https://www.eclipse.org/che/docs/che-7/overview/importing-certificates-to-browsers/
    ✔ Eclipse Che 7.20.0-SNAPSHOT has successfully installed.
    ✔ Users can access the Dashboard at   : https://che-che.192.168.99.100.nip.io
    ✔ Plugins can be viewed at            : https://plugin-registry-che.192.168.99.100.nip.io/v3
    ✔ Devfiles can be viewed at           : https://devfile-registry-che.192.168.99.100.nip.io
    ✔ Identity provider can be accessed at: https://keycloak-che.192.168.99.100.nip.io/auth
    ✔ Product documentation is at         : https://www.eclipse.org/che/docs/
    ✔ Eclipse Che admin credentials       : "admin:admin". You will be asked to change default Che admin password on the first login.
    ✔ Identity Provider credentials       : "admin:qzwHgv9ZkHdP"
```

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/18019

### How to test this PR?
1. Build the project
2. Deploy Eclipse Che on any platform
3. Check the output

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
